docs/001_develop/03_client-capabilities/008_interaction/criteria-segmented-control.mdx

```diff
@@ -0,0 +1,241 @@
+          <criteria-segmented-control criteriaOptions={{criteriaOptions}} onClick={handleCriteriaChange}>
+            <label slot="label">Select option</label>
+          </criteria-segmented-control>
+
+        );
+      }
+    ```
+  </TabItem>
+  <TabItem value="angular">
+
+    Declaration
+    ```html
+      <criteria-segmented-control></criteria-segmented-control>
+    ```
+
+    Usage
+    ```typescript
+      import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+      import { FormsModule } from '@angular/forms';
+      @Component({
+        selector: 'my-root',
+        template: `
+              <criteria-segmented-control
+                [criteriaOptions]="criteriaOptions"
+                (change)="handleCriteriaChange($event)">
+                <label slot="label">Filter by instrument name</label>
+              </criteria-segmented-control>
+            `,
+        standalone: true,
+        schemas: [CUSTOM_ELEMENTS_SCHEMA],
+        imports: [FormsModule],
+      })
+      export class AppComponent {
+
+        const criteriaOptions = [
+          { label: 'A', field: 'CHOSEN_OPTION', value: 'option-A', serialiser: Serialisers.EQ },
+          { label: 'B', field: 'CHOSEN_OPTION', value: 'option-B', serialiser: Serialisers.EQ },
+          { label: 'C', field: 'CHOSEN_OPTION', value: 'option-C', serialiser: Serialisers.EQ },
+        ];
+
+        handleCriteriaChange(event: Event) {
+          const target = event.target as HTMLInputElement;
+          console.log(target.value);
+        }
+      }
+      ```
+    </TabItem>
+  </Tabs>
+
+Criteria options are JSON objecst constructed using serialisers
+
+The choice of serialisers are:
+
+```typescript
+    Serialisers.AND
+    Serialisers.OR
+    Serialisers.NOT
+    Serialisers.GT
+    Serialisers.GE
+    Serialisers.LT
+    Serialisers.LE
+    Serialisers.NE
+    Serialisers.EQ
+    Serialisers.equals
+    Serialisers.equalsIgnoreCase
+    Serialisers.contains
+    Serialisers.fieldContains
+    Serialisers.containsIgnoreCase
+    Serialisers.dateIsEqual
+    Serialisers.dateIsGreaterEqual
+    Serialisers.dateIsLessEqual
+```
+
+
+## API
+
+### Properties:
+
+<table>
+  <thead><tr><th>Name</th><th>Type</th><th>Description</th><th>Example</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>criteriaOptions</td>
+      <td><code>array</code></td>
+      <td>Array of criteria objects that can be selected</td>
+<td>
+
+```typescript
+<criteria-segmented-control criteriaOptions=${(x) => x.sampleCriteriaOptions)}>
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>value</td>
+      <td><code>string</code></td>
+      <td>The value of the constructed criteria expression</td>
+<td>
+
+```typescript
+<criteria-segmented-control value="sampleValue">
+```
+
+</td>
+    </tr>
+    <tr>
+      <td>selectedLabel</td>
+      <td><code>string</code></td>
+      <td>The selected options label</td>
+<td>
+
+```typescript
+<criteria-segmented-control selectedLabel="selectedLabel">
+```
```

Comment:
Hold off on merging this PR because I am a bit confused about this property, I have messaged Michal about it.

Reply:
Double checked with Michal and Szymon and I think we can go with something like this
```suggestion
      <td>The selected options label. If you only need to work with the value for a form then use the `value` property, but you can use this property to set the default selected value, or set the selection when rehydrating state.</td>
<td>

```typescript
<criteria-segmented-control :selectedLabel=${(x) => defaultSelected}>
```
```